app/views/hosts/_list.html.erb

```diff
@@ -24,7 +24,7 @@
         <td class="hidden-xs ellipsis"><%= (icon(host.operatingsystem, :size => "18x18") + " #{host.operatingsystem.to_label}").html_safe if host.operatingsystem %></td>
         <td class="hidden-xs ellipsis"><%= host.environment %></td>
         <td class="hidden-tablet hidden-xs ellipsis"><%= model_name host %></td>
-        <td class="hidden-tablet hidden-xs"><%= label_with_link host.hostgroup, 26, @hostgroup_authorizer %></td>
+        <td class="hidden-tablet hidden-xs"><%= label_with_link host.hostgroup, 23, @hostgroup_authorizer %></td>
```

Comment:
This is a temporary fix to prevent the column from overflowing. Once #2137 finally gets merged we will get larger columns and be able to increase this. I am also looking into making this overflow in js like the rest of the columns, the issue here is that nested hostgroups should be ellipsized on the parent if possible and not on the hostname itself.